### DISCUSSION TITLE
Prepare for release v1.0.0

### DISCUSSION THREAD

**Discussion by gftea** [gftea/amqprs#25] [General]
I feel the current library is ready for **v1.0.0** release, waiting for user feedbacks from **v1.0.0-beta.0**

Once we release v1.0.0, we need to handle back-compatibility seriously and make sure there is good tracking of all the release contents

**Comment by michaelklishin** [comment]
Congrats on shipping 1.0! This is my favorite Rust client 👏

**Reply by gftea** [reply]
Thanks, Michael!
